CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ set(lib_name ${PROJECT_NAME})
 
 add_library(${lib_name} STATIC ${c_src})
 
-target_compile_definitions(${lib_name} PUBLIC __64BIT__)
+target_compile_definitions(${lib_name} PUBLIC)
```

Comment:
This line does nothing. Please see:

https://github.com/NOAA-EMC/NCEPLIBS-g2c/pull/28/commits/3f7ed54a03415c66e20dd86c79b7b6073b4ae2fa

Reply:
Dusan, 
I agree you can remove this line in "target_compile_definitions(${lib_name} PUBLIC)" in CMakeLists.txt.